utils/happo.js

```diff
@@ -35,7 +48,7 @@ const allExamples = requireContext
       // eslint-disable-next-line react/display-name
       variants[id] = () => (
         <ThemeProvider>
-          <LiveProvider code={source} scope={scope} mountStylesheet>
+          <LiveProvider code={source} scope={scope} mountStylesheet={false}>
```

Comment:
Ah, didn't know we could turn that off and still have it apply styles.

Reply:
Yep, that stylesheet is just for the default prism syntax highlighting in the code examples.  We implement our own custom syntax highlighting separately.